### PR TITLE
fix: bug fixes for #508

### PR DESCRIPTION
**Pull request by JMGordon**
Bug fixes for #508.

#### Changelog

**New**

- N/A

**Changed**

- Renamed child prop `closeMenu` to `onClick` in `OverflowMenu`.

- In `Tabs`, removed `onSelectionChange` from the props rest variable to eliminate it being passed into the `nav` element as an event handler. Reworked methods `handleTabClick`, `handleTabKeyDown`, and `handleTabAnchorFocus`, to receive `onSelectionChange` as an argument.

- Changed test `getTriggerPosition` in `Tooltip-test` so that an object is passed to `triggerPosition`, instead of a string.

**Removed**

- N/A

### CODE REVIEW

File: src/components/Tabs/Tabs.js [hunk 1]
```diff
@@ -97,13 +103,13 @@ export default class Tabs extends React.Component {
...
 
-  selectTabAt = index => {
+  selectTabAt = (index, onSelectionChange) => {
     if (this.state.selected !== index) {
       this.setState({
         selected: index,
       });
-      if (this.props.onSelectionChange) {
-        this.props.onSelectionChange(index);
+      if (onSelectionChange !== undefined) {
```
Comment (asudoh) [inline src/components/Tabs/Tabs.js]: Just a little thing - If we’d do more strict check than is-truthy, I’d recommend `typeof f === 'function'`.

File: src/components/Tooltip/Tooltip-test.js [hunk 2]
```diff
@@ -162,19 +162,33 @@ describe('Tooltip', () => {
     });
   });
 
-  describe('getTriggerPosition', () => {
+  describe.skip('getTriggerPosition', () => {
```
Comment (asudoh) [inline src/components/Tooltip/Tooltip-test.js]: Is `.skip` here intentional?

**Comment by asudoh** [comment]
Nice fix @JMGordon!

File: src/components/Tabs/Tabs.js (hunk 1 above)
Comment (JMGordon) [reply src/components/Tabs/Tabs.js]: Good idea. I'll switch out the conditional with the recommended check.

File: src/components/Tooltip/Tooltip-test.js (hunk 2 above)
Comment (JMGordon) [reply src/components/Tooltip/Tooltip-test.js]: Whoops... I forgot to remove the skip. Will remove on next commit.

**Pull request merged**